apps/docs/content/components/data.jsx

```diff
@@ -0,0 +1,34 @@
+export const installMethods = [
```

Comment:
I can see this file growing with time could this be act as an index loading diverse data?

Reply:
Or you can create an index to load the data if it grows.